modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/v3_7_3/UpgradeDDMFormInstanceReport.java

```diff
@@ -70,16 +69,25 @@ public UpgradeDDMFormInstanceReport(
 	protected void doUpgrade() throws Exception {
 		runSQL("delete from DDMFormInstanceReport;");
 
-		StringBundler sb1 = new StringBundler(12);
+		StringBundler sb1 = new StringBundler(21);
 
 		sb1.append("select DDMContent.data_, ");
-		sb1.append("DDMFormInstanceRecord.formInstanceRecordId from ");
-		sb1.append("DDMContent inner join DDMFormInstanceRecordVersion on ");
+		sb1.append("DDMFormInstanceRecord.formInstanceRecordId, ");
+		sb1.append("DDMStructureVersion.definition from DDMContent inner ");
+		sb1.append("join DDMFormInstanceRecordVersion on ");
 		sb1.append("DDMContent.contentId = ");
 		sb1.append("DDMFormInstanceRecordVersion.storageId inner join ");
 		sb1.append("DDMFormInstanceRecord on ");
 		sb1.append("DDMFormInstanceRecord.formInstanceRecordId = ");
-		sb1.append("DDMFormInstanceRecordVersion.formInstanceRecordId where ");
+		sb1.append("DDMFormInstanceRecordVersion.formInstanceRecordId join ");
```

Comment:
@rodrigopaulino, let's use `inner join` to keep consistency with existent query code. 🙂

Reply:
You got it.